ShopPricingGlossary.ttl

```diff
@@ -369,18 +369,17 @@ source:
   a skos:Concept ;
   wdrs:describedby source: ;
   skos:inScheme <http://www.openlinksw.com/data/turtle/general/ShopPricingGlossary#this> ;
-  schema:name "Maximum Concurrent Client Machines" ;
-  schema:comment """Maximum Concurrent Client Machines"""@en ;
-  schema:description """The maximum number of client hosts (physical or virtual, metered by IP address) which may simultaneously open
-                        ODBC, JDBC, ADO.NET, OLE&nbsp;DB, and/or XMLA database sessions through a Request Broker and/or Database or
-                        Bridge Agent under the given license. Only applicable to Enterprise Edition."""@en .
+  schema:name "Maximum Client Machine Installations" ;
+  schema:comment """Maximum Client Machine Installations"""@en ;
+  schema:description """The maximum client machines the ODBC, JDBC, ADO.NET, OLE&nbsp;DB, and/or XMLA client components can be installed on. 
+                          Only applicable to Enterprise Edition."""@en .

```

Comment:
The new naming and description are not accurate!

We have always permitted Generic Client drivers and providers to be installed on infinite machines, and we should not change that.

The existing `schema:description` is accurate.  `schema:name` and `schema:comment` might be made more clear, as I suggest here --

Therefore -- 

```suggestion
  schema:name "Maximum Concurrently Connected Client Machines"@en ;
  schema:comment """Maximum Concurrently Connected Client Machines"""@en ;
  schema:description """The maximum number of client hosts (physical or virtual, metered by IP address) which may simultaneously open
                        ODBC, JDBC, ADO.NET, OLE&nbsp;DB, and/or XMLA database sessions through a Request Broker and/or Database or
                        Bridge Agent under the given license. Only applicable to Enterprise Edition."""@en .
```